docs/source/version-5-configuration.md

```diff
@@ -1,71 +1,97 @@
+
+## `LOGGING_LEVEL`
+The logging level that can be set to one of `fatal`,
   `error`, `warn`, `info`, `debug`, or `trace`. (Default `info`)
-- `STATIC_URI` - The URI that static assets can be accessed from. This URI can
-  be to a proxy that uses this Coral server on `PORT` as the upstream. Disabled
-  by default.
-- `DISABLE_TENANT_CACHING` - When `true`, all tenants will be loaded from the
-  database when needed rather than keeping a in-memory copy in sync via
-  published events on Redis. (Default `false`)
-- `DISABLE_MONGODB_AUTOINDEXING` - When `true`, Coral will not perform indexing
-  operations when it starts up. This can be desired when you've already
-  installed Coral on the target MongoDB, but want to improve start performance.
-  **You should not use this parameter unless you know what you're doing! Upgrades
-  may introduce additional indexes that the application relies on.**
-  (Default `false`)
-- `LOCALE` - Specify the default locale to use for all requests without a locale
-  specified. (Default `en-US`)
-- `ENABLE_GRAPHIQL` - When `true`, it will enable the `/graphiql` even in
-  production, use with care. (Default `false`)
-- `CONCURRENCY` - The number of worker nodes to spawn to handle web traffic,
-  this should be tied to the number of CPU's available. (Default
-  `os.cpus().length`)
-- `DEV_PORT` - The port where the Webpack Development server is running on.
-  (Default `8080`)
-- `METRICS_USERNAME` - The username for _Basic Authentication_ at the `/metrics` and `/cluster_metrics`
-  endpoint.
-- `METRICS_PASSWORD` - The password for _Basic Authentication_ at the `/metrics` and `/cluster_metrics`
+
+## `DISABLE_CLIENT_ROUTES` 
+Disables mounting of client routes for developing with Webpack Dev Server (Default `false`)
+
+## `DISABLE_FORCE_SSL`
+Disables forcing SSL in production environments. Should not be used except for testing (Default `false`)
+
+## `DISABLE_LIVE_UPDATES` 
+When `true`, disables subscriptions for the comment
+stream for all stories across all tenants (Default `false`)
+
+## `DISABLE_MONGODB_AUTOINDEXING` 
+When `true`, Coral will not perform indexing operations when it starts up. This can be desired when you've already installed Coral on the target MongoDB, but want to improve start performance.
+  
+**You should not use this parameter unless you know what you're doing! Upgrades may introduce additional indexes that the application relies on.**
+(Default `false`)
+
+## `DISABLE_RATE_LIMITERS` 
+Disables the rate limiters in development. This will only work when also set to a development environment (Default `false`)
+
+## `DISABLE_TENANT_CACHING`
+When `true`, all tenants will be loaded from the database when needed rather than keeping a in-memory copy in sync via published events on Redis. (Default `false`)
+
+## `ENABLE_GRAPHIQL` 
+When `true`, it will enable the `/graphiql` even in
+  production, **use with care**. (Default `false`)
```

Comment:
It looks like additional line breaks were added, check to see if this was added in other places as well.

Reply:
This was intentional, as I changed all the formatting on this page to match how our v4 Configuration & Advanced Config pages were organized.

Reply:
I mean it is currently this:

```
When `true`, it will enable the `/graphiql` even in
  production ...
```

When it could just be:

```
When `true`, it will enable the `/graphiql` even in production ...
```

Reply:
I see, I actually thought you were referring to diff line breaks

Reply:
Corrected, and pushed changes!